subprojects/tooling-api/tooling-api.gradle

```diff
@@ -1,3 +1,5 @@
+import org.gradle.ShadedJar
```

Comment:
It's funny how quickly these things creep in when something isn't strict enough. `org.gradle` is implicitly imported, so this class was resolved, even though it is not part of the Gradle DSL, but from our `buildSrc` project.

Reply:
👀